### PR TITLE
Feature/remove agency

### PR DESCRIPTION
**Pull request by fceller**
Replaced the transaction / AgencyCache by agency call

### CODE REVIEW

**Review by joerg84 — APPROVED** [review]

File: debugging [hunk 1]
```diff
@@ -0,0 +1 @@
+lib
```
Comment (mchacki) [inline debugging:1]: What is this about?
It looks like a non-intentional include in this commit, or do I miss something?

**Review by mchacki — CHANGES_REQUESTED** [review]
The code-change looks good to me.
But there seems to be some helper file included.
Please Remove this, then GoodToGo

File: debugging (hunk 1 above)
Comment (fceller) [reply debugging:1]: this is just a symbloc link

Comment (mchacki) [reply debugging:1]: ah, did not recognize this, thanks!

**Review by mchacki — APPROVED** [review]
LGTM

**Pull request merged**